datascience/tables.py

```diff
@@ -5719,12 +5719,48 @@ def asdict(self):
             return collections.OrderedDict(zip(self._table.labels, self))
 
     class Rows(collections.abc.Sequence):
-        """An iterable view over the rows in a table."""
+        """
+        An iterable row-styled view of the table object that is passed as an argument.
+
+        args:
+        table- accepts a table object of class Table()
+
+        >>>  t = td.Table().with_columns({
+             'letter': ['a', 'b', 'c', 'z'],
+             'count':  [  9,   3,   3,   1],
+             'points': [  1,   2,   2,  10],
+        })
+
+        >>> td.Table().Rows(t)
+        Rows(letter | count | points
+        a      | 9     | 1
+        b      | 3     | 2
+        c      | 3     | 2
+        z      | 1     | 10)
+        """
         def __init__(self, table):
             self._table = table
             self._labels = None
 
         def __getitem__(self, i):
+            """
+            Access the i-th row of the given table.
+
+            args:
+            i- index of the Row that needs to be accessed.
+
+            returns:
+            Returns a Row object containing the i-th row of the given table
+
+            >>> t = td.Table().with_columns({
+                'letter': ['a', 'b', 'c', 'z'],
+                'count':  [  9,   3,   3,   1],
+                'points': [  1,   2,   2,  10],
+            })
+
+            >>> td.Table().Rows(t).__getitem__(0)
```

Comment:
You can reuse objects from previous docs. Let's save one Rows object in the class initializer and reuse that for the remaining methods in this class.